rootfs/etc/logrotate

```diff
@@ -5,8 +5,10 @@
         notifempty
         compress
         sharedscripts
+        prerotate
+            pkill goaccess
+        endscript
         postrotate
             if [ -s /usr/local/nginx/logs/nginx.pid ]; then nginx -s reload; fi
-            kill "$(pgrep goaccess)"
```

Comment:
since goaccess is forced restarted through "while true; do", it will be restarted before the logs are rotated and then crashed again, because the log file was recreated

Reply:
It's been removed from the while true do. It's only run now when the logrotate fires.

Reply:
yes and then crash because the log file was rotated

Reply:
I'm not seeing the crash you mention in my logs with the patch in place.

Reply:
if goaccess gets killed, it will be restarted instantly. But if the log rotation is slower, then the log file goaccess bound again, is removed, which will crash goaccess again

Reply:
I've added a 15 second delay. This will allow the log to rotate and Nginx to (re)start even on the slowest machines.